lib/pxclient.js

```diff
@@ -57,6 +64,25 @@ class PxClient {
         activity.details = details;
         return activity;
     }
+    async fetchRemoteConfig(config) {
+        try {
+            const callData = {
+                url: `https://sapi-${config.px_app_id}.perimeterx.net/config/`,
+                headers: { 'Authorization': `Bearer ${config.px_remote_config_secret}`, 'Accept-Encoding': '' },
+                timeout: 20000,
+            };
+            const res = await p({ url: callData.url, headers: callData.headers, timeout: callData.timeout, method: 'GET' });
+            const remoteConfigObject = JSON.parse(res.body);
+            return {
+                px_remote_config_id: remoteConfigObject.id,
+                px_remote_config_version: remoteConfigObject.version,
+                ... remoteConfigObject.configValue
+            };
+        } catch (e) {
+            this.sendRemoteLog(`Error fetching remote configurations: ${e.message}`, LoggerSeverity.DEBUG, 'write_remote_config', config);
```

Comment:
- extract `Error fetching remote configurations: ${e.message}` to `const message` for better readability
- you have a TypeError for errorType, use it instead of `"write_remote_config"`
- Remove `return undefined` it doesn't contribute anything, in JS function always returns undefined which makes adding `return undefined` a noise code.